### PR TITLE
Add Radio and RadioGroup to react-components/unstable

### PR DESCRIPTION
**Pull request by behowell**
## New Behavior

* Export Radio, RadioGroup, and related types from `@fluentui/react-components/unstable`.
* Make `@fluentui/react-radio` public.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `171.671 kB`<br />`48.071 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |   `32.519 kB`<br />`10.64 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/a0aef79692f29d01a09a4415a59eb50db3bf163c'>a0aef79692f29d01a09a4415a59eb50db3bf163c</a></sub>
<div id="bundleSizeComment9423"/>

**Review by bsunderhus — APPROVED** [review]

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/FluentProviderWithTheme-mount_base.html">149</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/FluentProviderWithTheme-mount.html">182</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/FluentProviderWithTheme-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Avatar-mount_base.html">759</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Avatar-mount.html">758</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/BaseButton-mount_base.html">724</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/BaseButton-mount.html">808</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Breadcrumb-mount_base.html">2277</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Breadcrumb-mount.html">2248</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ButtonNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/ButtonNext-mount_base.html">399</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/ButtonNext-mount.html">415</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Checkbox-mount_base.html">1269</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Checkbox-mount.html">1279</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/CheckboxBase-mount_base.html">1119</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/CheckboxBase-mount.html">1056</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/ChoiceGroup-mount_base.html">3922</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/ChoiceGroup-mount.html">3963</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/ComboBox-mount_base.html">865</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/ComboBox-mount.html">822</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/CommandBar-mount_base.html">8797</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/CommandBar-mount.html">8939</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/ContextualMenu-mount_base.html">7203</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/ContextualMenu-mount.html">7473</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/DefaultButton-mount_base.html">985</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/DefaultButton-mount.html">938</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/DetailsRow-mount_base.html">3110</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/DetailsRow-mount.html">3203</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/DetailsRowFast-mount_base.html">3157</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/DetailsRowFast-mount.html">3203</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/DetailsRowNoStyles-mount_base.html">2987</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/DetailsRowNoStyles-mount.html">2927</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Dialog-mount_base.html">1947</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Dialog-mount.html">2023</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/DocumentCardTitle-mount_base.html">151</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/DocumentCardTitle-mount.html">168</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Dropdown-mount_base.html">2771</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Dropdown-mount.html">2775</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/FluentProviderNext-mount_base.html">1744</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/FluentProviderNext-mount.html">1655</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/FluentProviderWithTheme-mount_base.html">149</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/FluentProviderWithTheme-mount.html">182</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/FluentProviderWithTheme-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/FluentProviderWithTheme-virtual-rerender_base.html">81</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/FluentProviderWithTheme-virtual-rerender.html">92</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">175</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount.html">222</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/FocusTrapZone-mount_base.html">1619</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/FocusTrapZone-mount.html">1504</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/FocusZone-mount_base.html">1517</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/FocusZone-mount.html">1516</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/IconButton-mount_base.html">1514</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/IconButton-mount.html">1492</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Label-mount_base.html">325</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Label-mount.html">297</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Layer-mount_base.html">2495</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Layer-mount.html">2409</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Link-mount_base.html">414</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Link-mount.html">411</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/MakeStyles-mount_base.html">1500</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/MakeStyles-mount.html">1432</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/MenuButton-mount_base.html">1290</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/MenuButton-mount.html">1310</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/MessageBar-mount_base.html">1711</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/MessageBar-mount.html">1774</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Nav-mount_base.html">2651</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Nav-mount.html">2863</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/OverflowSet-mount_base.html">980</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/OverflowSet-mount.html">893</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Panel-mount_base.html">1832</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Panel-mount.html">1658</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Persona-mount_base.html">756</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Persona-mount.html">702</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Pivot-mount_base.html">1187</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Pivot-mount.html">1255</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/PrimaryButton-mount_base.html">1129</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/PrimaryButton-mount.html">1066</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Rating-mount_base.html">6536</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Rating-mount.html">6187</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/SearchBox-mount_base.html">1114</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/SearchBox-mount.html">1108</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Shimmer-mount_base.html">2200</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Shimmer-mount.html">2127</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Slider-mount_base.html">1646</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Slider-mount.html">1586</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/SpinButton-mount_base.html">4236</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/SpinButton-mount.html">4278</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Spinner-mount_base.html">351</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Spinner-mount.html">360</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/SplitButton-mount_base.html">2718</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/SplitButton-mount.html">2724</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Stack-mount_base.html">474</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Stack-mount.html">469</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/StackWithIntrinsicChildren-mount_base.html">1948</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/StackWithIntrinsicChildren-mount.html">2009</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/StackWithTextChildren-mount_base.html">4467</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/StackWithTextChildren-mount.html">4500</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/SwatchColorPicker-mount_base.html">9579</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/SwatchColorPicker-mount.html">9718</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/TagPicker-mount_base.html">2283</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/TagPicker-mount.html">2264</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/TeachingBubble-mount_base.html">11143</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/TeachingBubble-mount.html">11249</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Text-mount_base.html">388</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Text-mount.html">394</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/TextField-mount_base.html">1225</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/TextField-mount.html">1222</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/ThemeProvider-mount_base.html">1000</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/ThemeProvider-mount.html">1004</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/ThemeProvider-virtual-rerender_base.html">559</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/ThemeProvider-virtual-rerender.html">551</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1523</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1652</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Toggle-mount_base.html">688</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/Toggle-mount.html">684</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/buttonNative-mount_base.html">131</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21883/perf-test/buttonNative-mount.html">142</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Review by micahgodbolt — APPROVED** [review]

**Review by spmonahan — APPROVED** [review]

File: packages/react-radio/package.json [hunk 1]
```diff
@@ -1,8 +1,7 @@
 {
   "name": "@fluentui/react-radio",
   "version": "9.0.0-beta.1",
```
Comment (khmakoto) [inline packages/react-radio/package.json:3]: Should we change this version to 9.0.0-rc as well?

Comment (ecraig12345) [reply packages/react-radio/package.json:3]: No, if it's unstable it's still beta.

**Pull request merged**